airflow/providers/openlineage/utils/utils.py

```diff
@@ -395,3 +397,10 @@ def wrapper(*args, **kwargs):
             log.exception(e)
 
     return wrapper
+
+
+def is_source_enabled():
```

Comment:
`from airflow.compat.functools import cache` and then 
```suggestion
@cache
def is_source_enabled():
```

Reply:
done.